### ISSUE TITLE
Deprecation: move stylesheets to `styles/` directory

### ISSUE THREAD

**Issue by braver** [carlo/rainbow-theme#1]
Via deprecation cop:

Store package style sheets in the `styles/` directory instead of `stylesheets/` in the `git-control` package

```
ThemePackage.getStylesheetsPath (/Applications/Atom.app/Contents/Resources/app/src/package.js:434:9)
PackageWatcher.watch (/Applications/Atom.app/Contents/Resources/app/node_modules/dev-live-reload/lib/package-watcher.js:44:35)
```

**Comment by czottmann** [comment]
"`git-control` package"?

Your point is valid, but I'm confused by your mention of said package as I am not its maintainer and this is not its repo.

**Comment by braver** [comment]
Well that's weird. This is a report straight from the UI in atom. Didn't see the reference to git-control, but that just doesn't make any sense. Anyway... :confused:

**Comment by czottmann** [comment]
¯\＿(ツ)＿/¯

Either way, I'll be fixing it soon!

**Comment by czottmann** [comment]
Fixed in [v1.5.0](/carlo/rainbow-theme/tree/v1.5.0).

**Comment by braver** [comment]
:+1:
